packages/composite-checkout/src/theme.js

```diff
@@ -0,0 +1,37 @@
+/* @format */
```

Comment:
Our prettier setup now formats without these format pragmas so they can be removed.